pkg/pocket/executor/common.go

```diff
@@ -116,3 +125,37 @@ func (e *Executor) TxnRollback() error {
 	}
 	panic("unhandled txn rollback switch")
 }
+
+// since the tiflash cluster is always conn1 in TiFlash abtest, so it is
+// okay to just use single test ddl here.
+func (e *Executor) createTiFlashTableReplica(table string) error {
+	return e.SingleTestExecDDL(fmt.Sprintf("ALTER TABLE %s SET TIFLASH REPLICA 1", table))
+}
+
+func (e *Executor) WaitTiFlashTableSync(table string) error {
+	sql := fmt.Sprintf("SELECT AVAILABLE FROM information_schema.tiflash_replica WHERE TABLE_SCHEMA = '%s' and TABLE_NAME = '%s'",
+		e.dbname, table)
+	for {
+		res, err := e.GetConn().Select(sql)
+		if err != nil {
+			return err
+		}
+		// table doesn't exist
+		if res == nil {
+			return nil
```

Comment:
if table doesn't exist, is returning nil ok?

Reply:
Not sure. This is the case for `s4` and `s5`, I am not really sure how to deal with that. Since these tables are not really creating, so it will always be nil.